AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/general.py

```diff
@@ -25,3 +25,63 @@
     "inactive": 1,
     "editor": 2,
 }
+
+
+class Strings:
+    EMPTY_STRING = ""
+    ONLY_SPACE = " "
+    ESCAPED_SPACE = "\n"
+    CHARACTER = "a"
+    NUMBER = '1'
+    SHORT_STRING = "ShortString"
+    STRING_WITH_SPACE = "String With Space"
```

Comment:
nit: Should this be `STRING_WITH_SPACES` (or change the string to just have one space)

Reply:
question: I'm not as familiar with Python syntax but are these effectively 'static' members (not per instance) and also `const`/immutable? Just want to check they can't be modified anywhere else if possible.

Reply:
they are only a naming convention as constant. they are actually mutable but by convention all cap names imply constant that you should not change

Reply:
Unfortunately Python doesn't have support for actual immutable Constants. As a result, there has been a stylization adopted of using all caps with underscores between words.  This is also the convention suggested by PEP-8.

https://peps.python.org/pep-0008/#constants

Reply:
Updated the string to match the name intent.